db/seeds/abilities/edit_posts.html

```diff
@@ -0,0 +1,9 @@
+<h2>What does this ability allow me to do?</h2>
+<p>This ability allows you to unilaterally edit posts, as well as review suggested edits from other users.</p>
+
+<p>Edits should be used to improve posts, including edits improving spelling, grammar, and formatting, as well as fixes for missing <a href="/help/alt-text">alt text</a>, broken images and links, and other improvements. Edits should, however, respect the goal of the original poster.</p>
+
+<img src="https://meta.codidact.com/uploads/PW9TNLAZwQWfaX1RMCBEbHDt" alt="Edit post button">

```

Comment:
Any images in these should be included in this PR and linked as other included assets (usually an `/assets/xxx` path)